ui_vue/cypress/e2e/signin.cy.ts

```diff
@@ -0,0 +1,22 @@
+describe('Sign In', () => {
+
+  it('wrong credentials', () => {
+    cy.visit('/sign-in');
+    cy.url().should('include', '/sign-in');
+    cy.get('[data-cy="email"]', { timeout: 10000 }).type('testuser06@gmail.com');
+    cy.get('[data-cy="password"]').type('Abcd2067*');
+    cy.get('[data-cy="submit"]').click();
+    cy.url().should('include', '/sign-in');
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
```

Comment:
Why disabled ESLint?

Reply:
I wanted to add some time between the two tests but eslint was giving error of unnecessary wait.
I added this time gap to observe proper ui changes during testing.